CorePush/Utils/AppleCryptoHelper.cs

```diff
@@ -0,0 +1,51 @@
+﻿
+using System;
+using System.Security.Cryptography;
+
+#if NETSTANDARD2_0
```

Comment:
@danielgindi I don't mind using BouncyCastle for all builds. Would you be able to make a change. No need for conditional compiling here. There are quite a few issues with out of the box crypto.

Reply:
For use of bouncy castle you just need to revert your latest commit that removed it... But I can update this one as this introduces a few syntax changes to adhere to .net standard 2.0.

So are you sure you want to revert to BouncyCastle on all targets?

Reply:
Now you can choose between this one and #45 :-)